src/pyhf/cli/spec.py

```diff
@@ -108,7 +108,7 @@ def inspect(workspace, output_file, measurement):
     )
 
     # summary
-    fmtStr = '{{0: >{0:d}s}}  {{1:s}}'.format(maxlen + len('Summary'))
+    fmtStr = '{{: >{:d}s}}  {{:s}}'.format(maxlen + len('Summary'))
```

Comment:
Oops, this is a Ruff mistake, should be reported! :)

Reply:
Oh, actually, I think it isn't. Only one of these is a substitution (the second 0), and the resulting string then will have a 0 and a 1 - which will then work. So Ruff is correct.